.github/workflows/github-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Create GitHub Release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
Can we be more restrictive with something like `v[0-9]+\.[0-9]+\.[0-9]+\.*` to avoid creating releases on a tag like `"verbose"`?